roadmap/implementors-guide/guide.md

```diff
@@ -1043,6 +1045,24 @@ I have a basic implementation of this code on the [`ashley-test-statement-gossip
 
 ----
 
+## Gossip Protocols
+
+A lot of a relay chain's communication happens via gossiping messages. To avoid this becoming an attack vector, we want to use [Bounded Gossip Protocols](https://github.com/w3f/research/blob/master/docs/polkadot/networking/0-overview.md#bounded-gossip-protocols).
+
+### Functionality
+
+Several components make up a gossiping system:
+
+* A network implementation, that contains things like the actual connections to peers.
+* A propagation pool of messages.
+
+When the node wants to gossip a message:
+* The message is put into the propagation pool and stays there until cleaned up.
+* The message is sent out to the nodes that it is currently connected to.
+
+When a new node connects to our node:
+* All messages in the propagation pool are sent to it, and we receive all of its propagation pool messages.
```

Comment:
Same here - we only send messages to a peer that pass the filtration-criterion `allowed_k`. At this level of detail, we can paraphrase that as a function `is_message_allowed(peer, msg)` which gossip protocols need to implement